src/client/index.js

```diff
@@ -160,11 +161,10 @@ export class BncClient {
    * @return {Promise}
    */
   async setPrivateKey(privateKey, localOnly = false) {
-    if (privateKey !== this.privateKey) {
+    if (privateKey !== this._privateKey) {
       const address = crypto.getAddressFromPrivateKey(privateKey, this.addressPrefix)
       if (!address) throw new Error(`address is falsy: ${address}. invalid private key?`)
-      if (address === this.address) return this // safety
```

Comment:
Check of `address` has been removed, since `setPrivateKey` wont work by setting a private key again if it has been removed by `removePrivateKey()` before. As a reminder: `removePrivateKey()` has been added with this PR here.

Reply:
@sectore This lib is gradually changed from js to ts. Can you submit a new to the ts branch? Thanks a lot

Reply:
@xzjcool ^ Done (see #265)